README.md

```diff
@@ -96,29 +103,25 @@ See [Contributing.md](.github/Contributing.md)
 - Autopilot goes direct to RWY on APP (same with the default A320) [Workaround: Use DIR to a waypoint or selected heading]
 - Upper ECAM displays wrong THR levers position / N1 rating.
 - Newly added ASOBO A320 liveries are incompatible with the A32NX mod.
-- Ruder keybindings not working (you have to set your keybinding to rudder axis right and left)
+- Rudder keybindings not working (you have to set your keybinding to rudder axis right and left)
 - Wing dips on landing (due to bad transition to direct law in flare, same with the default A320)  [Workaround use minimal aileron input on landing]
-- Black screens / unable to start (conflict with another mod/livery or incorrect installation of the A32NX mod, use the [installer](https://api.flybywiresim.com/installer))
+- Black screens/unable to start (conflict with another mod/livery or incorrect installation of the A32NX mod, use the [installer](https://api.flybywiresim.com/installer))
 
 ## License
 
-The original contents of this repository are DUAL LICENSED.
-
-Original textual-form source code in this repository is licensed under the GNU General Public License version 3. Artistic assets (including but not limited to images, sound banks, 3D models, and textures) are licensed under [Creative Commons CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/).
-
-Any other original works found to be not explicitly covered by the above definitions are all rights reserved to avoid ambiguities at the moment, but we are very permissive about their use if you contact us.
+The original contents of this repository are licensed under the MIT license.
 
 **Microsoft Flight Simulator © Microsoft Corporation. The FlyByWire Simulations A32NX was created under Microsoft's "Game Content Usage Rules" using assets from Microsoft Flight Simulator, and it is not endorsed by or affiliated with Microsoft.**
 
 ## FAQ
 
 **Q: Can I download the aircraft in the current state?**
 
-A: Yes, see [Downloads](#Downloads).
+A: Yes, see [downloads.](https://github.com/flybywiresim/a32nx#downloads)
 
 **Q: Where is the plane in-sim?**
 
-A: The mod is added on top of the default A320neo from Asobo. If you want to uninstall it and revert to the original aircraft, just delete the A32NX folder in your community folder.
+A: The mod is now separate from the base A320neo aircraft and will show up as a new aircraft under the name "FlyByWire Simulations A320neo (LEAP)". If you want to uninstall it and revert to the original aircraft, just delete the A32NX folder in your community folder (paths for all editions are above).

```

Comment:
I would just remove this section

Reply:
Honestly, we should just move the FAQ to the docs site. Most end-users don't get that far down on GitHub, our front end solutions (website or docs site) would serve this better.